invenio_files_rest/config.py

```diff
@@ -127,3 +129,10 @@
 
 FILES_REST_FILE_TAGS_HEADER = 'X-Invenio-File-Tags'
 """Header for updating file tags."""
+
+FILES_REST_OFFLOAD_STREAMING_EXTERNALLY = False
+"""Use the X-Accel-Redirect header to stream the file through NGINX"""
+
+FILES_REST_STREAMING_REDIRECT_RESPONSE_FUNC =  \
```

Comment:
Continuing my previous comment, what about if we for something similar to:
```
FILES_REST_XSENDFILE_ENABLED = False

FILES_REST_XSENDFILE_RESPONSE_FUNC = ...
```
not sure about the second though, we should check in Invenio what we are using....